proteus/mprans/GN_SW2DCV.h

```diff
@@ -17,6 +17,7 @@ namespace py = pybind11;
 #define REESTIMATE_MAX_EDGE_BASED_CFL 0
 #define LAMBDA_MGN 1
 #define IF_BOTH_GAMMA_BRANCHES 0
+#define LIMITING_ITERATION 2
```

Comment:
out of curiosity what's the significance of stopping at the second iteration?

Reply:
Well the limiting helps preserve positivity of the water height when we use a higher-order (in space) numerical method. The idea is that you can apply the limiting however many times you'd like, but we do it twice as a safe guard. Besides more computations, there shouldn't be a difference between doing it 2 or 10 times.